tests/deployment/serverless/test_aws_lambda.py

```diff
@@ -0,0 +1,38 @@
+import os
+from ruamel.yaml import YAML
+
+from bentoml.deployment.serverless.aws_lambda import (
+    generate_aws_lambda_serverless_config,
+    generate_aws_lambda_handler_py)
+
+
+def test_generate_aws_lambda_serverless_config(tmpdir):
+    python_version = '3.7.2'
+    deployment_name = 'test_deployment_lambda'
+    api_names = ['predict']
+    region = 'us-west-test'
+    namespace = 'namespace'
+    generate_aws_lambda_serverless_config(
```

Comment:
nit: let's make generate_aws_lambda_serverless_config return the path to the generated config file?